ProcessesApi/Startup.cs

```diff
@@ -57,6 +57,7 @@ public Startup(IConfiguration configuration)
         public void ConfigureServices(IServiceCollection services)
         {
             services.AddCors();
+            services.AddApiGateway();
```

Comment:
Where is this method defined?

Reply:
This is from Hackney.Core.HTTP, Mike added it just before he left!